zeebe/atomix/cluster/src/main/java/io/atomix/cluster/AtomixCluster.java

```diff
@@ -286,15 +297,23 @@ public String toString() {
   }
 
   /** Builds a default messaging service. */
-  protected static ManagedMessagingService buildMessagingService(final ClusterConfig config) {
+  protected static ManagedMessagingService buildMessagingService(
+      final ClusterConfig config, final String schedulerPrefix) {
     return new NettyMessagingService(
-        config.getClusterId(), config.getNodeConfig().getAddress(), config.getMessagingConfig());
+        config.getClusterId(),
+        config.getNodeConfig().getAddress(),
+        config.getMessagingConfig(),
+        schedulerPrefix);
   }
 
   /** Builds a default unicast service. */
-  protected static ManagedUnicastService buildUnicastService(final ClusterConfig config) {
+  protected static ManagedUnicastService buildUnicastService(
+      final ClusterConfig config, final String schedulerPrefix) {
     return new NettyUnicastService(
-        config.getClusterId(), config.getNodeConfig().getAddress(), config.getMessagingConfig());
+        config.getClusterId(),
+        config.getNodeConfig().getAddress(),
+        config.getMessagingConfig(),
+        schedulerPrefix);
```

Comment:
:x: Same here.

Reply:
As previously mentioned that is being done in the `AtomixClusterConfiguration` constructor.